in_toto/keylib_test.go

```diff
@@ -141,3 +141,68 @@ k7Gtvz/iYzaLrZv33cFWWTsEOqK1gKqigSqgW9T26wO9AgMBAAE=
 		}
 	}
 }
+
+func testParseEd25519FromPrivateJSON(t *testing.T) {
```

Comment:
Judging from the coverage report, these tests didn't run. 
`s/test/Test/` seems to do the trick. :)